.travis.yml

```diff
@@ -12,3 +12,6 @@ before_script:
   - sh -c "if [ $REDIS_STANDALONE -eq 0 ]; then cd phpredis-2.2.3/ && phpize && ./configure && make && make install; fi"
   - sh -c "if [ $REDIS_STANDALONE -eq 0 ]; then echo \"extension=redis.so\" >> `php --ini | grep \"Loaded Configuration\" | sed -e \"s|.*:\s*||\"`; fi"
   - composer install
+script:
+  - ./vendor/bin/phpcs --standard=PSR2 src/ test/
```

Comment:
Might be useful to wrap this up in a [composer script](https://getcomposer.org/doc/articles/scripts.md).